test/__snapshots__/validate-options.test.js.snap.webpack4

```diff
@@ -18,6 +18,16 @@ exports[`options validate should throw an error on the "client" option with '{"h
    -> Tells clients connected to devServer to use the provided host."
 `;
 
+exports[`options validate should throw an error on the "client" option with '{"hotEntry":[""]}' value 1`] = `
+"ValidationError: Invalid configuration object. Object has been initialized using a configuration object that does not match the API schema.
+ - configuration.client.hotEntry should be one of these:
+   boolean | function
```

Comment:
I think we don't need this to be `function`, because `client` will be `function`, but maybe we need do it in the next PR because we don't want to break tests (we will rewrite them in the next PR), right?

Reply:
Yes